src/l1-resources/stream-instance/README.md

```diff
@@ -0,0 +1,69 @@
+# stream-instance
+
+The official [MongoDB Atlas](https://www.mongodb.com/) AWS CDK resource for Node.js.
+
+> AWS CDK [L1 construct] and data structures for the [AWS CloudFormation Registry] type `MongoDB::Atlas::StreamInstance`.
+[L1 construct]: https://docs.aws.amazon.com/cdk/latest/guide/constructs.html
+[AWS CloudFormation Registry]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/registry.html
+
+## Description
+
+Returns, adds, edits, and removes stream instances.
```

Comment:
We should call out in all CDK documentation as well that this is in public preview like we did in CFN. 

Also, we should at least include reference to this README https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/examples/atlas-streams/README.md 
cc @Zuhairahmed in case he wants to add anything.

Reply:
I added the same as we did in CFN to make explicit that it's in preview.

Reply:
I was also referring to the https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/master/examples/atlas-streams/README.md because it also mentions about Stream processors.

We need to make sure all relevant documentations points from the Scope doc are addressed.